packages/compass/src/app/index.js

```diff
@@ -224,22 +222,16 @@ var Application = View.extend({
       this.tourClosed();
     }
   },
-  showOptIn: function() {
-    if (process.env.HADRON_ISOLATED !== 'true') {
```

Comment:
How would this currently work in Isolated edition? Would we display settings that don’t have any effect on users?

Reply:
we hide the menu item in ([menu.ts](https://github.com/mongodb-js/compass/pull/3222/files#diff-94ecbcd9e84609b4725e01f12f7f8c01168f17639ffff65dc8f6f27ef11a4cddR336)). and as a follow up, we also need to add `@mongodb-js/compass-settings` to `compass/src/app/plugins/default.js`

Reply:
The `tourClosed()` function belowed used this one’s logic though – right now it always broadcasts `window:show-network-optin`

Reply:
nice, i'll add this check there.